.bazelrc

```diff
@@ -1,7 +1,7 @@
 # To update these lines, execute
 # `bazel run @contrib_rules_bazel_integration_test//tools:update_deleted_packages`
-build --deleted_packages=examples/cc/external,examples/command_line/external,examples/ios_app,examples/ios_app/CoreUtilsObjC,examples/ios_app/Example,examples/ios_app/ExampleNestedResources,examples/ios_app/ExampleObjcTests,examples/ios_app/ExampleResources,examples/ios_app/ExampleTests,examples/ios_app/ExampleUITests,examples/ios_app/OnlyStructuredResources,examples/ios_app/TestingUtils,examples/ios_app/Utils,examples/ios_app/external,examples/ios_app/test/fixtures,examples/ios_app/test/fixtures/bwb.xcodeproj/rules_xcodeproj/links,examples/ios_app/test/fixtures/bwx.xcodeproj/rules_xcodeproj/links,App.xcodeproj/rules_xcodeproj/bazel,App.xcodeproj/rules_xcodeproj/links
```

Comment:
@cgrindel Seems like there should be a better way for the glob to ignore `bazel-output-base`. Also, it doesn't seem to work:
```
ERROR: infinite symlink expansion detected
[start of symlink chain]
/Users/administrator/buildbuddy/remote_build/dc1af18c-964f-41ec-b641-f925f65f273a/output-base/execroot/com_github_buildbuddy_io_rules_xcodeproj/bazel-out/darwin-fastbuild/bin/examples/ios_app_test.runfiles/com_github_buildbuddy_io_rules_xcodeproj
/Users/administrator/buildbuddy/remote_build/dc1af18c-964f-41ec-b641-f925f65f273a/output-base/execroot/com_github_buildbuddy_io_rules_xcodeproj/bazel-out/darwin-fastbuild/bin/examples/ios_app_test.runfiles/com_github_buildbuddy_io_rules_xcodeproj/examples/ios_app/bazel-output-base/external/com_github_buildbuddy_io_rules_xcodeproj
[end of symlink chain]
```

Reply:
I'll see if I can adjust where I set it to get around this.

Reply:
Got around it with some `.bazelignore` entires. It still has loads more entries than needed.

Reply:
I have run into symlink issues after running a build manually in the child workspace and then running a build in the parent workspace. I have worked around the issue by running `bazel clean --expunge` in the child workspace.

For some background, the reason we use the `--deleted_packages` mechanism instead of `.bazelignore` is [lightly document here](https://github.com/bazel-contrib/rules_bazel_integration_test/blob/bf73bdb085525f5e6059e89af30a6b749f1f3802/README.md#4-configure-the-deleted-packages-for-the-parent-workspace). My recollection is `glob()` will not find files under ignored packages. The following is an excerpt from the doc:

> For glob() in the parent workspace to find files in the child workspaces, we need to use the deleted packages mechanism.

Is the issue discussed in this PR related to using an Xcode project in the child workspace?

Reply:
> Is the issue discussed in this PR related to using an Xcode project in the child workspace?

No, it was related to using an in-repo output_base. The only way around that was `.bazelignore`. 

The biggest issue though is that this `--deleted_packages` setting is massive now. I could manually adjust it (I think, to just list the ignore output base and it should cover child packages, right?), but then we lose the use of the tool to update it.

Reply:
Gotcha. The `--deleted_packages` mechanism is very clunky. Unfortunately, it is not recursive. Each package needs to be listed. 😞

For now, we could write a script that adds the stuff that you have been doing manually. Alternatively, if the `ios_app` example is causing more trouble than it is worth, we could revert it to exist like the other examples.

Reply:
Currently Workflows don't support sub-directories, but once they do, I think I'll convert our integration test to just be a Workflow action rooted in the example directory, removing the need for `
rules_bazel_integration_test`.